packages/preset-bootstrap/src/index.js

```diff
@@ -215,7 +215,7 @@ export const bootstrap = {
   sizes,
   shadows,
   radii,
-  typeStyles,
```

Comment:
The components use `text`, not `textStyles` as a key FWIW

Reply:
I've fixed it for pulse and bootstrap themes. There're plenty of textStyles keys in another themes tough.